functions/src/resolve-market.ts

```diff
@@ -74,6 +78,20 @@ export const resolvemarket = newEndpoint(opts, async (req, auth) => {
   const creator = await getUser(creatorId)
   if (!creator) throw new APIError(500, 'Creator not found')
 
+  if (contract.outcomeType === 'FREE_RESPONSE') {
+    if (resolutions) {
+      Object.keys(resolutions).forEach((outcome) =>
+        validateFreeResponseOutcome(contract, outcome)
+      )
+      const pctSum = sum(Object.values(resolutions))
+      if (pctSum !== 100) {
```

Comment:
This is going to be incorrect due to floating point error.

Reply:
Fixed, using a tolerance of 0.1 now.